### ISSUE TITLE
Paddle support hinge loss？

### ISSUE THREAD

**Issue by 20092136** [PaddlePaddle/Paddle#6096]
Cannot find content or API about hinge loss in Paddle‘s official web, is paddle supports it?
If not, how could I implement it? THx~~

**Comment by sweetsky0901** [comment]
抱歉暂时还没有支持 在我们计划的新版本中会支持的

**Comment by 20092136** [comment]
假设我用hingeloss作pairwise ranking，最后想实现

loss = max(0, margin - label * (pos_score-neg_score))

这样是不是只要用 layer_math.py 求一下而不需要修改C++就可以呢？

**Comment by qingqing01** [comment]
@20092136 

理论是可以用 `relu` + ` layer_math.py`里的函数去实现，但是作为loss的，计算backward时，loss层输入的梯度(output_grad)需要设置为1，在PaddlePaddle的layer里，cost层相比其他layer都是特殊实现，这里用`relu`和其他math layer组合hinge loss，最后一层会是`relu`，应该是不行的，backward的计算会不对，其他暂时没想到不写代码可替代的方法。 

可以自己实现，也可以提出要求，让PaddlePaddle同学实现。

**Comment by chengduoZH** [comment]
该issue长期没有更新，暂时先关闭了，如果需继续讨论再打开
